source/hardware/controllers/hercules_djcontrol_MIX.rst

```diff
@@ -0,0 +1,116 @@
+Hercules DJControl MIX
+==============================
+
+-  `Manufacturer’s product page <https://www.hercules.com/en-us/product/djcontrol-mix/>`__
+-  `Manufacturer’s support and downloads page <https://support.hercules.com/en/product/djcontrolmix-en/>`__
+-  `Forum thread <https://mixxx.discourse.group/t/hercules-contrl-mix-mapping/26581>`__
+
+.. versionadded:: 2.3
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI, so it can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows.
+
+.. note::
+   In order to use this mapping, the DJControl MIX must be physically connected to the computer through USB. It was not conceived to use the controller connected via Bluetooth.
+
+Sound Card Setup
+----------------
+
+This device does not have a built in audio interface.
+
+Please refer to :ref:`the user manual <controllers-without-an-integrated-audio-interface>` for more details about the audio configuration in Mixxx.
```

Comment:
This link target does not exist.

Reply:
Should point to this page : [https://manual.mixxx.org/2.3/en/chapters/example_setups#controllers-without-an-integrated-audio-interface]

Reply:
I think you need to reference this anchor (which is placed wrong in the example setups chapter by the way ):
https://github.com/mixxxdj/manual/blob/d309759da9fd9f9b7471d06d2ff134fc30245b6e/source/chapters/example_setups.rst?plain=1#L164

Reply:
Just made the change

Reply:
The syntax is not a correct reference

Reply:
Hopefully time time will be correct.